website/docs/api/architectures.mdx

```diff
@@ -924,6 +924,15 @@ plausible [`Candidate`](/api/kb/#candidate) objects. The default
 `CandidateGenerator` uses the text of a mention to find its potential aliases in
 the `KnowledgeBase`. Note that this function is case-dependent.
 
+### spacy.CandidateBatchGenerator.v1 {id="CandidateBatchGenerator"}
```

Comment:
This part doesn't seem related to the rest of the PR? Or did you mean to document `spacy.EmptyKB.v2` instead?

Reply:
Ah, I meant to move that into a separate PR. I noticed that `spacy.CandidateBatchGenerator.v1` hasn't been documented yet.

Reply:
[PR](https://github.com/explosion/spaCy/pull/12350).